src/services/event-producer/sqs/sqs-event-producer-service.ts

```diff
@@ -0,0 +1,40 @@
+import 'reflect-metadata'
+import { Config } from 'node-config-ts'
+import { inject, singleton } from 'tsyringe'
+import { SQSClient, SendMessageCommand } from '@aws-sdk/client-sqs'
+import { EventProducerService } from '../event-producer-service.js'
+import { v4 as uuidv4 } from 'uuid'
+
+@singleton()
+export class SQSEventProducerService implements EventProducerService {
+  private readonly sqsClient: SQSClient
+  constructor(@inject('config') private config?: Config) {
+    this.sqsClient = new SQSClient({
+      region: process.env.AWS_REGION,
```

Comment:
@stephhuynh18 AWS library gets it from process.env by its own, no? Are the env variables here for visibility?

Reply:
:o i didn't know that i'll remove this config thanks